spec/controllers/emancipations_controller_spec.rb

```diff
@@ -14,12 +14,12 @@
 
```

Comment:
This and the following spec were changed when I ran `bundle exec standardrb --fix`